core/src/main/scala/org/apache/gearpump/jarstore/FileServer.scala

```diff
@@ -145,8 +145,7 @@ object FileServer {
     }
 
     private def entity(file: File)(implicit ec: ExecutionContext): Future[RequestEntity] = {
-      val entity = HttpEntity(MediaTypes.`application/octet-stream`, file.length(),
-        FileIO.fromFile(file, chunkSize = 100000))
+      val entity = HttpEntity(MediaTypes.`application/octet-stream`, file, chunkSize = 100000)
```

Comment:
yes its unconnected ,shall remove it if you feel so ,
However was wondering if it made the method more focused & better performant on setting up entity with file directly using Akka http API than us us doing a FileIO ?
The akka http API does the same plus 
1-  we don't fetch the length of the file ourselves ,
2- short-circuits if the file length were zero , prevent graph orchestration possibly for File IO reads of empty files.

Believe these can be useful for a client context.

Let me know otherwise , shall undo.